demisto_sdk/commands/validate/tests/IN_validators_test.py

```diff
@@ -5963,3 +5966,458 @@ def test_IsValidFeedExpirationPolicy_fully_fetched_feed_with_suddenDeath():
+                        ]
+                    ],
+                )
+            ],
+            [
+                create_integration_object(
+                    paths=["configuration"],
+                    values=[
+                        [
+                            {
+                                "name": "param1",
+                                "required": True,
+                                "defaultvalue": "default",
+                            }
+                        ]
+                    ],
+                )
+            ],
+            1,
+            [
+                'Possible backward compatibility break: You have added the following new *required* parameters: "param2", "param3". Please undo the changes or provide '
+                "default values. If cannot give default value, please make the parameter not "
+                "required and check in code implementation that it was configured."
+            ],
+        ),
+        # Case 10: Invalid - changed from non-required to required without default
+        (
+            [
+                create_integration_object(
+                    paths=["configuration"],
+                    values=[
+                        [
+                            {
+                                "name": "param1",
+                                "required": True,
+                                "defaultvalue": "default",
+                            },
+                            {
+                                "name": "param2",
+                                "required": True,
+                            },  # Changed from false to true with no default
+                        ]
+                    ],
+                )
+            ],
+            [
+                create_integration_object(
+                    paths=["configuration"],
+                    values=[
+                        [
+                            {
+                                "name": "param1",
+                                "required": True,
+                                "defaultvalue": "default",
+                            },
+                            {"name": "param2", "required": False},
+                        ]
+                    ],
+                )
+            ],
+            1,
+            [
+                'Possible backward compatibility break: You have added the following new *required* parameters: "param2". Please undo the changes or provide '
+                "default values. If cannot give default value, please make the parameter not "
+                "required and check in code implementation that it was configured."
+            ],
+        ),
+    ],
+)
+def test_IsNewRequiredParamNoDefaultIntegrationValidator_obtain_invalid_content_items(
+    content_items, old_content_items, expected_number_of_failures, expected_msgs
+):
+    """
+    Given
+    content_items and old_content_items iterables.
```

Comment:
let's add the cases here instead of above.

Reply:
done